Packs/SimpleEmailReputation/Integrations/SimpleEmailReputation/SimpleEmailReputation.yml

```diff
@@ -0,0 +1,48 @@
+category: Data Enrichment & Threat Intelligence
+commonfields:
+  id: Simple Email Reputation
+  version: -1
+configuration:
+- defaultvalue: https://emailrep.io/
+  display: 'Server URL '
+  name: url
+  required: true
+  type: 0
+- display: API Key
+  name: apikey
+  required: true
+  type: 4
+- defaultvalue: "false"
+  display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+- defaultvalue: "false"
+  display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+description: |-
+  EmailRep is a system of crawlers, scanners and enrichment services that collects data on email addresses, domains, and internet personas.
+
+  EmailRep uses hundreds of data points from social media profiles, professional networking sites, dark web credential leaks, data breaches, phishing kits, phishing emails, spam lists, open mail relays, domain age and reputation, deliverability, and more to predict the risk of an email address.
+display: Simple Email Reputation
+name: Simple Email Reputation
+script:
+  commands:
+  - arguments:
+    - default: true
+      description: A List of email addresses - comma separated
+      isArray: true
+      name: emailrep-get-reputation
+      required: true
+    description: Return email information and reputation
+    name: email
```

Comment:
Seems like you have something wrong here. Your command name is `email` but the argument name is `emailrep-get-reputation`. please change it and make sure that your command works on demisto as expected.